docs/release-notes/2.33.0.md

```diff
@@ -0,0 +1,145 @@
+# Percona Monitoring and Management 2.33.0
+
+| **Release date:** | Dec 13, 2022                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+!!! caution alert alert-warning "Important"
+    We recommend using the latest version of PMM. This ensures that you have access to the latest PMM features and that your environment runs on the latest version of the underlying components, such as VictoriaMetrics, with all the bug fixes in place.
+
+## Release Highlights
+
+
+### High Availability in PMM
```

Comment:
What about using this as the header instead of "High Availibility in PMM" ?
Prevent data loss with VM integration for short outages

Reply:
@dygksy I do not completely agree with this. It's a part of HA. We want this to be highlighted right? Morever, we are mentioning it will be released in stages...